module/PowerShellEditorServices/Commands/Public/Set-ScriptExtent.ps1

```diff
@@ -55,41 +57,16 @@ function Set-ScriptExtent {
             }
         }
 
-        foreach ($aExtent in $extentList) {
+        foreach ($kvp in $extentList.GetEnumerator()) {
+            $aExtent = $kvp.Value
             $aText = $Text
 
             if ($needsIndentFix) {
-                # I'd rather let PSSA handle this when there are more formatting options.
                 $indentOffset = ' ' * ($aExtent.StartColumnNumber - 1)
-                $aText = $aText -split '\r?\n' `
-                                -join ([Environment]::NewLine + $indentOffset)
+                $aText = $aText -split '\r?\n' -join ([Environment]::NewLine + $indentOffset)
```

Comment:
This is just formatting. The rest of this should probably be revisited at some point as well, but not actively bad afaik